### DISCUSSION TITLE
finger keep open and stiff after click init in diagnostic tools

### DISCUSSION THREAD

**Discussion by yani-rl-ai** [HumanSupportRobot/QandA#81] [Q&A]
i just solved my problem in this discussion
https://github.com/orgs/HumanSupportRobot/discussions/71
when i click init in diagnostic tools web, the finger in hsr gripper could not back into default position and when i turn off hsr it keep stiff and hard. it looks like stuck.  how to solve it ?

**Comment by hsr-admin** [comment]
Please remove the cover on the side of the hand and open the hand by turning to the loosening direction using a hexagonal wrench (nominal size 6).

(reference)
https://docs.hsr.io/hsrb_user_manual_en/howto/replace_hand_parts.html#id2
